java/configuration/src/main/java/sleeper/configuration/properties/instance/QueryProperty.java

```diff
@@ -80,6 +80,13 @@ public interface QueryProperty {
                     "value given below is 128KiB. This value can be overridden using the query config.")
             .defaultValue("" + (128 * 1024)) // 128 KiB
             .propertyGroup(InstancePropertyGroup.QUERY).build();
+    UserDefinedInstanceProperty DEFAULT_QUERY_WARM_LAMBDA_EXECUTION_PERIOD_IN_MINUTES = Index
```

Comment:
To keep consistent with other default properties (which can be overridden on a per-table basis), could we rename this to `QUERY_WARM_LAMBDA_EXECUTION_PERIOD_IN_MINUTES`

Reply:
Property remianed